src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Input/TappedTapped_Tests.cs

```diff
@@ -122,6 +122,7 @@ public void When_InListViewWithItemClick()
 
 		[Test]
 		[AutoRetry]
+		[ActivePlatforms(Platform.Android, Platform.iOS)] // Disabled on WASM: False failure
```

Comment:
Still valid ?

Reply:
Unfortunately yes ☹ This has been detailed here https://github.com/unoplatform/uno/issues/2739#issuecomment-714156433